src/handlers/inbound/accounts/{ID}.ts

```diff
@@ -0,0 +1,83 @@
+ and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed
+ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+  - Sridhar Voruganti - sridhar.voruganti@modusbox.com
+
+ --------------
+ ******/
+
+import {
+  thirdparty as tpAPI
+} from '@mojaloop/api-snippets'
+import { Message } from '~/shared/pub-sub'
+import {
+  InboundAccountsModel,
+  InboundAccountsModelConfig
+} from '~/models/inbound/accounts.model'
+import {
+  OutboundAccountsModel
+} from '~/models/outbound/accounts.model'
+import { Request, ResponseObject } from '@hapi/hapi'
+import { StateResponseToolkit } from '~/server/plugins/state'
+
+/**
+ * Handles a inbound PUT /accounts/{ID} request
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+async function put (_context: any, request: Request, h: StateResponseToolkit): Promise<ResponseObject> {
+  const payload = request.payload as unknown as tpAPI.Schemas.AccountsIDPutResponse
+  const userId: string = request.params.ID
+  const channel = OutboundAccountsModel.notificationChannel(userId)
+  const pubSub = h.getPubSub()
+  // don't await on promise to resolve, let finish publish in background
+  pubSub.publish(channel, payload as unknown as Message)
+  return h.response().code(200)
+}
+
+/**
+ * Handles a inbound GET /accounts/{ID} request
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+async function get (_context: any, request: Request, h: StateResponseToolkit): Promise<ResponseObject> {
+  const userId: string = request.params.ID
+  const logger = h.getLogger()
+  const sourceFspId = request.headers['fspiop-source']
+  const config: InboundAccountsModelConfig = {
+    logger,
+    backendRequests: h.getBackendRequests(),
+    thirdpartyRequests: h.getThirdpartyRequests()
+  }
+  const model = new InboundAccountsModel(config)
+  const response = await model.getUserAccounts(userId, sourceFspId)
```

Comment:
this is blocking Switch until model.getUserAccounts promise is resolved
setImmediate should be used to not block `h.response().code(202)`

```suggestion
setImmediate(async () => {
  const response = await model.getUserAccounts(userId, sourceFspId)
  logger.push({ response }).info('InboundAccountsModel handled GET /accounts/{ID} request')
})
```

Reply:
Fixed